From 3d23ec7f0857cbc1ae8e42bdd27ab179a7e8344f Mon Sep 17 00:00:00 2001
From: "Hong Jen Yee (PCMan)" <pcman.tw@gmail.com>
Date: Fri, 19 Apr 2019 01:19:08 +0800
Subject: [PATCH] Fix the license header of code we took from libfm (they
 should use LGPL2 instead of GPL2). - src/core/vfs/vfs-menu.c -
 src/core/vfs/vfs-search.c These files are taken from libfm which has been
 relicensed to LGPL 2 on 2014-09-05 by Andrey N. Gritsenko after consulting
 other contributors.

---
 src/core/vfs/vfs-menu.c   | 21 ++++++++++-----------
 src/core/vfs/vfs-search.c | 37 +++++++++++++++++--------------------
 2 files changed, 27 insertions(+), 31 deletions(-)

diff --git a/src/core/vfs/vfs-menu.c b/src/core/vfs/vfs-menu.c
index c541159..069b1bc 100644
--- a/src/core/vfs/vfs-menu.c
+++ b/src/core/vfs/vfs-menu.c
@@ -4,20 +4,19 @@
  *
  *      Copyright 2012-2014 Andriy Grytsenko (LStranger) <andrej@rep.kiev.ua>
  *
- *      This program is free software; you can redistribute it and/or modify
- *      it under the terms of the GNU General Public License as published by
- *      the Free Software Foundation; either version 2 of the License, or
- *      (at your option) any later version.
+ *      This library is free software; you can redistribute it and/or
+ *      modify it under the terms of the GNU Lesser General Public
+ *      License as published by the Free Software Foundation; either
+ *      version 2.1 of the License, or (at your option) any later version.
  *
- *      This program is distributed in the hope that it will be useful,
+ *      This library is distributed in the hope that it will be useful,
  *      but WITHOUT ANY WARRANTY; without even the implied warranty of
- *      MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
- *      GNU General Public License for more details.
+ *      MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ *      Lesser General Public License for more details.
  *
- *      You should have received a copy of the GNU General Public License
- *      along with this program; if not, write to the Free Software
- *      Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston,
- *      MA 02110-1301, USA.
+ *      You should have received a copy of the GNU Lesser General Public
+ *      License along with this library; if not, write to the Free Software
+ *      Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
  */
 
 #ifdef HAVE_CONFIG_H
diff --git a/src/core/vfs/vfs-search.c b/src/core/vfs/vfs-search.c
index 0a64fb6..92b932c 100644
--- a/src/core/vfs/vfs-search.c
+++ b/src/core/vfs/vfs-search.c
@@ -1,25 +1,22 @@
 /*
- * fm-vfs-search.c
- * 
- * Copyright 2012 Hong Jen Yee (PCMan) <pcman.tw@gmail.com>
- * Copyright 2010 Shae Smittle <starfall87@gmail.com>
- * 
- * This program is free software; you can redistribute it and/or modify
- * it under the terms of the GNU General Public License as published by
- * the Free Software Foundation; either version 2 of the License, or
- * (at your option) any later version.
- * 
- * This program is distributed in the hope that it will be useful,
- * but WITHOUT ANY WARRANTY; without even the implied warranty of
- * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
- * GNU General Public License for more details.
- * 
- * You should have received a copy of the GNU General Public License
- * along with this program; if not, write to the Free Software
- * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston,
- * MA 02110-1301, USA.
- * 
+ *      fm-vfs-search.c
  * 
+ *      Copyright 2012 Hong Jen Yee (PCMan) <pcman.tw@gmail.com>
+ *      Copyright 2010 Shae Smittle <starfall87@gmail.com>
+ *
+ *      This library is free software; you can redistribute it and/or
+ *      modify it under the terms of the GNU Lesser General Public
+ *      License as published by the Free Software Foundation; either
+ *      version 2.1 of the License, or (at your option) any later version.
+ *
+ *      This library is distributed in the hope that it will be useful,
+ *      but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *      MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ *      Lesser General Public License for more details.
+ *
+ *      You should have received a copy of the GNU Lesser General Public
+ *      License along with this library; if not, write to the Free Software
+ *      Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
  */
 
 #ifdef HAVE_CONFIG_H

